modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/SemanticServiceMetrics.java

```diff
@@ -191,6 +197,31 @@ private CachedMeters metersForEndpoint(String endpoint) {
            Optional.empty();
   }
 
+  /**
+   * Checks for latency-slo-metrics configuration options and sets up
+   * metrics that track how many requests meet a certain latency goal.
+   *
+   * @param id
+   * @param endpoint
+   * @return
+   */
+  private Optional<LatencySloTracker> requestLatencySloTracker(MetricId id, String endpoint) {
```

Comment:
is the endpoint just the name of the route or does it include the method too? would be nice to include that in the javadoc.

Reply:
It has to include the method